### PR TITLE
Add initial OTLP ingestion docs

### PR DESCRIPTION
**Pull request by gouthamve**
Partially addresses https://github.com/prometheus/prometheus/issues/12642

We still need a guide that we can link users to in https://github.com/prometheus/docs/tree/main/content/docs/guides This guide should show sending metrics from application directly via the OTel SDKs and also sending through the Collector.

### CODE REVIEW

File: docs/querying/api.md [hunk 1]
```diff
@@ -1294,3 +1294,16 @@ Enable the remote write receiver by setting
...
+## OTLP Receiver
+
+Prometheus can be configured as a receiver for the OTLP Metrics protocol. This 
+is not considered an efficient way of ingesting samples. Use it
+with caution for specific low-volume use cases. It is not yet suitable for
+replacing the ingestion via scraping yet.
+
+Enable the OTLP receiver by the feature flag
+`--enable-feature=otlp-write-receiver`. When enabled, the OTLP receiver
+endpoint is `/otlp/v1/metrics`.
```
Comment (songjiayang) [inline docs/querying/api.md:1307]: hi, the endpoint maybe `/api/v1/otlp/v1/metrics`.

File: docs/querying/api.md [hunk 2]
```diff
@@ -1294,3 +1294,16 @@ Enable the remote write receiver by setting
 endpoint is `/api/v1/write`. Find more details [here](../storage.md#overview).
 
 *New in v2.33*
+
+## OTLP Receiver
+
+Prometheus can be configured as a receiver for the OTLP Metrics protocol. This 
+is not considered an efficient way of ingesting samples. Use it
+with caution for specific low-volume use cases. It is not yet suitable for
```
Comment (roidelapluie) [inline docs/querying/api.md:1302]:
```suggestion
with caution for specific low-volume use cases. It is not suitable for
```

File: docs/querying/api.md [hunk 3]
```diff
@@ -1294,3 +1294,16 @@ Enable the remote write receiver by setting
...
+replacing the ingestion via scraping yet.
```
Comment (roidelapluie) [inline docs/querying/api.md:1303]:
```suggestion
replacing the ingestion via scraping.
```

File: docs/querying/api.md (hunk 1 above)
Comment (arukiidou) [reply docs/querying/api.md:1307]: you are right. 
code example

```go
otlphttp, err := otlpmetrichttp.New(
		ctx,
		otlpmetrichttp.WithInsecure(),
		otlpmetrichttp.WithEndpoint("localhost:9090"),
		otlpmetrichttp.WithURLPath("/api/v1/otlp/v1/metrics"),
)
```

**Review by roidelapluie — APPROVED** [review]
LGTM

**Pull request merged**